static/styles/bootstrap/bootstrap.css

```diff
@@ -3126,6 +3126,20 @@ table th[class*="span"],
   border-color: #ddd;
   border-color: rgba(0, 0, 0, 0.15);
 }
+.well-l {
+
+  min-height: 20px;
+  padding: 24px;
+  margin-bottom: 20px;
+  background-color: #f5f5f5;
+  border: 1px solid #e3e3e3;
+  -webkit-border-radius: 4px;
+     -moz-border-radius: 4px;
+          border-radius: 4px;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.05);
```

Comment:
@svineet pressed said in Ireland that changes can be made in bootstrap.css file that's why I edited it. 

@samsruti it will not have an effect on any of elements in any template  as I made a separate div class  for  it.